### PR TITLE
Issue/#41/add dev chat server instructions

### PR DESCRIPTION
**Pull request by aevo98765**
Adding a description for devs to setup a local model chat server. Two approaches have been provided:

1) Local setup of the ilab cli environment and spinning server up. We link users to the documentation for this.

2) Provide a docker image that can be pulled from dockerhub and used very easily. The issue with this approach is that the inference speed is very slow as Docker containers can't use mac GPUs at the moment.

Feedback very welcome on this one. Cheers!

### CODE REVIEW

File: server/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,21 @@
+FROM python:3.11
```
Comment (nerdalert) [inline server/Dockerfile:1]: Mind changing the file name to `Containerfile`

File: docs/development.md [hunk 2]
```diff
@@ -69,15 +69,62 @@ npm run pretty
...
+
+For the chat functionality to work you need a ilab model chat instance. To run this locally:
+
+`cd server`
+
+[https://github.com/instructlab/instructlab?tab=readme-ov-file#-getting-started](https://github.com/instructlab/instructlab?tab=readme-ov-file#-getting-started)
+
+After you use the `ilab serve` command you should have, by default, a chat server instance running on port 8000.
+
+### 2) Using docker
```
Comment (nerdalert) [inline docs/development.md:84]: Mind changing to Podman because of their Docker desktop licensing we are using Podman. Ty!

**Comment by nerdalert** [comment]
Good stuff, tyvm! Just swap from Docker to Podman because of the Docker Desktop licensing not being free anymore. Ty!

File: server/Dockerfile (hunk 1 above)
Comment (aevo98765) [reply server/Dockerfile]: Done

File: docs/development.md (hunk 2 above)
Comment (aevo98765) [reply docs/development.md]: Done

**Review by nerdalert — APPROVED** [review]
LGTM

**Pull request merged**